modules/xmpp/SDPUtil.js

```diff
@@ -595,6 +595,61 @@ const SDPUtil = {
             payloadTypes.unshift(payloadType);
             videoMLine.payloads = payloadTypes.join(' ');
         }
+    },
+
+    /**
+     * Strips the given codec from the given mline. All related RTX payload
+     * types are also stripped. If the resulting mline would have no codecs,
+     * it's disabled.
+     *
+     * @param {object} videoMLine the video mline object from an sdp as parsed
+     * by transform.parse.
+     * @param {string} codecName the name of the codec which will be stripped.
+     */
+    stripVideoCodec(videoMLine, codecName) {
```

Comment:
this is a good candidate for a unit test, i think.  i have one for the preferCodec already you could follow.

Reply:
True that! Will do.